### PR TITLE
feat: charity

### PR DESCRIPTION
**Pull request by guidiaz**

### CODE REVIEW

File: migrations/settings.js [hunk 1]
```diff
@@ -2,24 +2,48 @@ const { merge } = require("lodash")
...
-          reservePrice:   "1000000000000000000", //  1.00 ETH
-          startingPrice: "32000000000000000000", // 32.00 ETH
-          startingTs:     1677974400 + 7 * 3600, // Sun, 5 March 2023 0:00 GMT-7
-        }
+        launch: {
+          metadata: {
+            name: "ETHDenver 2023",
+            venue: "National Western $SPORK Castle",
+            startTs: 1677193200 + 7 * 3600, // Thu, 24 February 2023 0:00 GMT-7
+            endTs:   1677866400 + 7 * 3600, // Fri, 3 March 2023 18:00 GMT-7
```
Comment (aesedepece) [inline migrations/settings.js:17]: Let's run the game until the 4th, as the event finishes on the 5th!

File: migrations/settings.js [hunk 2]
```diff
@@ -2,24 +2,48 @@ const { merge } = require("lodash")
...
+            witnessReward: "1000000000", // 1.0 WIT
+            witnessCollateral: "15000000000", // 15.0 WIT
+          },
+        },
+        fractionalize: {
+          auctionSettings: {
+            deltaPrice:       "50000000000000000", //  0.05 ETH
+            deltaSeconds:                    3600, //     1 hour
+            reservePrice:   "1000000000000000000", //  1.00 ETH
+            startingPrice: "32000000000000000000", // 32.00 ETH
```
Comment (aesedepece) [inline migrations/settings.js:42]: Let's be bold here. I see no reason to start lower than 100 ETH.

File: migrations/settings.js [hunk 3]
```diff
@@ -2,24 +2,48 @@ const { merge } = require("lodash")
...
+            numWitnesses: 16,
+            minConsensusPercentage: 75,  // %
+            minerCommitFee: "100000000", // 0.1 WIT
+            witnessReward: "1000000000", // 1.0 WIT
+            witnessCollateral: "15000000000", // 15.0 WIT
+          },
+        },
+        fractionalize: {
+          auctionSettings: {
+            deltaPrice:       "50000000000000000", //  0.05 ETH
```
Comment (aesedepece) [inline migrations/settings.js:39]: If we go with a starting price of 100ETH, deltaPrice will need to be 0.15 so as to keep the auction runtime shorter than 1 month.

**Review by aesedepece — CHANGES_REQUESTED** [review]

File: migrations/settings.js (hunk 1 above)
Comment (aesedepece) [reply migrations/settings.js:17]: So 24h more would be `1677952800 + 7 * 3600`

**Review by Tommytrg — APPROVED** [review]

**Review by aesedepece — APPROVED** [review]

**Pull request merged**